app/views/confirmation_mailer/abuse.text.erb

```diff
@@ -0,0 +1,9 @@
+Sehr geehrte(r) Nutzer/-in von <%= Settings::Instance.name %>,
+
+um Ihre Missbrauchsmeldung zu bestätigen rufen Sie bitte folgende URL auf: <%= @url %>
+
+Diese URL funktioniert für <%= Jobs::Issue.deletion_deadline %> Stunden und kann einmalig aufgerufen werden. Nach Ablauf dieser Frist wird Ihre Missbrauchsmeldung automatisch hinfällig.
+
+Vielen Dank und mit freundlichen Grüßen
+
+Ihr <%= Settings::Instance.name %>-Team
```

Comment:
Mailer können auch eigene Layouts haben -> für ConfirmationMailer bitte ein Text-Layout erstellen in dem nur die Zeile 3 mit der Url der individuelle Content via `yield` eingefügt wird.